src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$month-length: 30;
+//powyżej wpisz długość miesiąca//
+$size: 100px;
+$gap: 1px;
+$padding: 10px;
+$days:
+  // odkomentuj pierwszy dzień miesiąca
+
+  // "mon" 0,
+  // "tue" 1,
+  // "wed" 2,
+  // "thu" 3,
```

Comment:
those should be modified in browser instead of code - please implement modifier - readme point 4